### PR TITLE
Updated sitemap to reflect all new pages

### LINKED ISSUE #262: Updates to Sitemap
Main sitemap has to be updated as number of pages have been changed. 

All pages in the root directory should be covered under this. 

P.S. There is a separate sitemap for blog and gyan which is maintained separately

### PR DESCRIPTION
**Pull request by salman-bhai**
Closes #262

### CODE REVIEW

File: sitemap-main.xml [hunk 1]
```diff
@@ -2,14 +2,17 @@
 <sitemapindex xmlns="https://www.sitemaps.org/schemas/sitemap/0.9">
 
 	<sitemap>
-    <loc>https://ieee.nitk.ac.in/sitemap.xml</loc>
+        <loc>https://ieee.nitk.ac.in/sitemap.xml</loc>
+		<changefreq>daily</changefreq>
```
Comment (mahim23) [inline sitemap-main.xml]: Is `changefreq` daily really needed? I think weekly or monthly would be more apt given how much we update the site. Wdyt?

Comment (rohit-mp) [reply sitemap-main.xml]: weekly for blog and monthly for everything else seems apt.

**Comment by salman-bhai** [comment]
@mahim23 have made the changes. Can you review once ?

**Comment by nimbus98** [comment]
@mahim23 @salman-bhai

**Comment by salman-bhai** [comment]
@nimbus98 @mahim23 add an approval. I'll do the merge post that if needed.

**Pull request merged**